pkg/cache/informer_cache.go

```diff
@@ -152,6 +153,25 @@ func (ic *informerCache) GetInformer(ctx context.Context, obj client.Object) (In
 	return i.Informer, err
 }
 
+func (ic *informerCache) getInformerForKind(ctx context.Context, gvk schema.GroupVersionKind, obj runtime.Object) (bool, *internal.Cache, error) {
+	var cache *internal.Cache
+	var started bool
+	var err error
+	switch ic.missPolicy {
+	case MissPolicyFail:
+		var ok bool
+		cache, started, ok = ic.Informers.Peek(gvk, obj)
```

Comment:
```suggestion
		if cache, started, ok := ic.Informers.Peek(gvk, obj); !ok {
```

Reply:
This won't set the `var cache, started` above though?

Reply:
```
 pkg/cache/informer_cache.go:161:10: indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (revive)
		} else {
			return started, cache, nil
		} 
```

your linter does not like this